lib/util/versioning.js

```diff
@@ -1,6 +1,26 @@
 const VID_SEP = require('arsenal').versioning.VersioningConstants
           .VersionId.Separator;
 
+/**
+ * @param {string} key full object key (_id in metadata)
+ * @returns {bool} true if key belongs to master object
+ */
+function isMasterKey(key) {
```

Comment:
i didn't add `isMasterKey` actually, it was already there i just changed the way it was written to be more consistent with the `extractVersionId` function. It was implemented directly in the `module.exports` object.
And i do agree, i don't see the use of it here as like you said there are no visible references in backbeat calling this function.

Reply:
I saw you didn't add it, but we can drive-by remove it and reexport it directly from Arsenal to reduce the amount of code duplication we have, since we're here anyway :wink: